apollo-router-core/Cargo.toml

```diff
@@ -25,6 +26,7 @@ once_cell = "1.9.0"
 router-bridge = { git = "https://github.com/apollographql/federation.git", rev = "950eb931e38746bb7cfed05382d6970a22e43cc4" }
 serde = { version = "1.0.133", features = ["derive", "rc"] }
 serde_json = { version = "1.0.74", features = ["preserve_order"] }
+serde_json_bytes = { version = "0.1.3", features = ["preserve_order"]}
```

Comment:
:grey_question: can't we get rid of `serde_json`?

Reply:
serde_json is still used in various places(like https://github.com/apollographql/router/pull/284/files#diff-3fee2c973320bfc619c5c548a725cfffbc3874e50bbd76351eca927d8f89ee6eR91), and is still imported by serde_json_bytes

Reply:
But this could probably be replaced by serde_json_bytes too, yes?

Reply:
probably, I will try

Reply:
so, serde_json cannot be removed because the `json!` macro, `from_str`, `to_string` are still used in multiple places. serde_json_bytes could reexport them, I'd rather have it provide a limited interface andd let serde_json handle the rest

Reply:
ok